### PR TITLE
feat: UKTS 40 - Add concept of airfield group

### PR DESCRIPTION
**Pull request by AxonC**
https://vatsimuk.atlassian.net/browse/UKTS-40

### CODE REVIEW

File: database/factories/AirfieldFactory.php [hunk 1]
```diff
@@ -10,5 +10,6 @@
         'icao' => (string) $faker->lexify('????'),
         'iata' => $faker->lexify('???'),
         'display_name' => $faker->city,
+        'airfield_group_id' => null,
```
Comment (atoff) [inline database/factories/AirfieldFactory.php:13]: Shouldn't need this

File: database/migrations/2020_03_27_131152_create_airfield_groups_table.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    public function up()
+    {
+        Schema::create('airfield_groups', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->string('name');
+        });
+
+        Schema::table('airfields', function (Blueprint $table) {
+            $table->unsignedBigInteger('airfield_group_id')->nullable();
+            $table->foreign('airfield_group_id')->references('id')->on('airfield_groups');
```
Comment (atoff) [inline database/migrations/2020_03_27_131152_create_airfield_groups_table.php:23]: hmm not a fan of these foreign keys, tend to muck everything up. Any particular reason we want this?

**Review by atoff — CHANGES_REQUESTED** [review]
Also, I'm not sure if the HasMany / belongsTo relationship is best here. Is there a case where we might want a belongsToMany relationship both ways so that an airfield can be in multiple groups?

File: database/migrations/2020_03_27_131152_create_airfield_groups_table.php (hunk 2 above)
Comment (AxonC) [reply database/migrations/2020_03_27_131152_create_airfield_groups_table.php:23]: Only if used incorrectly

File: database/factories/AirfieldFactory.php (hunk 1 above)
Comment (AxonC) [reply database/factories/AirfieldFactory.php:13]: wanted to make it clear. no sense in leaving it

**Comment by AxonC** [comment]
> 
> 
> Also, I'm not sure if the HasMany / belongsTo relationship is best here. Is there a case where we might want a belongsToMany relationship both ways so that an airfield can be in multiple groups?

For now, we are leaving a one-to-one mapping between a group and an airfield. Happy to change it in the future.

**Review by atoff — APPROVED** [review]

**Pull request merged**